styles/website.css

```diff
@@ -539,7 +546,7 @@ img {
 }
 @media (max-width: 1240px) {
     .markdown-section img {
-        max-width: 100%
+        max-width: 100%;
```

Comment:
Trailing semi-colons are actually optional (maybe not by the spec but probably with all relevant implementations), but this doesn't hurt.